src/doc/using_visit/Operators/OperatorTypes/DeferExpression_operator.rst

```diff
@@ -32,3 +32,14 @@ evaluated until after the ExternalSurface operator has been applied.
 
    DeferExpression operator example
 
+
+Deferring multiple expressions
+""""""""""""""""""""""""""""""
+
+What if you want to color a surface by a new variable equal to **(1.0 - X)^2** where **X** is the x-component of the surface's normal?
```

Comment:
I think we use bold (e.g. double stars as in `**`) for names of things like operators, gui windows, etc. Isn't it preferred to use double back tick instead of bold for these as in suggested for one case below

```suggestion
What if you want to color a surface by a new variable equal to ``(1.0 - X)^2`` where ``X`` is the x-component of the surface's normal?
```

Reply:
Thanks for the reminder. Done.